src/components/SortedList.js

```diff
@@ -1,14 +1,60 @@
 import React from 'react';
 import ListItem from './ListItem';
+import { filter as removePunctuation, isOutOfDate } from '../lib/helper';
 
 const SortedList = ({ itemsKeys, data }) => {
+  const compareItems = (howSoonA, howSoonB, nameA, nameB) =>
+    // compareItems() checks which item has the less number of the estimated days.
+    // returning -1 one for the item whose howSoon is less
+    // means that sort() will put it first and therefore comes first on the list
+    howSoonA < howSoonB
+      ? -1
+      : // the number with the greater number of the estimated days comes second
+      howSoonA > howSoonB
+      ? 1
+      : // reaching this line means that both items
+      // have the same number of estimated days until the next purchase.
+      // so now we have to compare by items name
+      // and since names are unique one name must be greater than the other
+      removePunctuation(nameA) > removePunctuation(nameB)
+      ? 1
+      : -1;
+
   return (
-    <ul>
+    <ul aria-label="List of items" className="sorted-list">
```

Comment:
no need to use aria here `ul` is already a semantic element

Reply:
Debora and I were thinking of removing the `<h1>List of Items</h`>` in the List route. So we might need this later but if we don't, yes, we can definitely remove it. :)

Reply:
What I mean is that this `aria-label` will be ignored by screen readers because `ul` is semantic enough if you tried it on screen readers it will give you something like "A list with 6 items"

Reply:
Ah, I see! I guess we won't need it then. I'll remove it. Thank you, Sarah. :))